ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/jboss/narayana/lra/main/module.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I can perhaps expect to find this module to be defined but I am a bit surprised that there was not already a module.xml for LRA?

Reply:
I think there was a single module, both for rest-at and lra under modules/system/layers/base/org/jboss/narayana/**rts**/main/module.xml
see https://github.com/wildfly/wildfly/tree/main/ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/jboss/narayana/rts/main @tomjenkinson